src/Sulu/Bundle/AdminBundle/Resources/js/components/Divider/divider.scss

```diff
@@ -0,0 +1,34 @@
+@import '../../containers/Application/colors.scss';
+
+$dividerBackgroundColor: $silver;
+
+.divider {
+    display: flex;
+    align-items: center;
+    font-size: 18px;
+    margin: 60px 0;
+
+    &::before,
+    &::after {
+        content: "";
+        display: inline-block;
+        flex: 1;
+        height: 1px;
+        background: $dividerBackgroundColor;
+    }
+
+    &::before {
+        margin-right: 80px;
```

Comment:
40px

Reply:
I have also seen 80px and 20px in the design 🙈